stix_shifter_modules/aws_athena/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -0,0 +1,3100 @@
+            "object": "creator-email-addr"
+          },
+          "groups": {
+            "desc": {
+              "key": "user-account.extensions.x-accessor-ext.group_desc",
+              "object": "creator-user-account"
+            },
+            "name": {
+              "key": "user-account.extensions.x-accessor-ext.group_name",
+              "object": "creator-user-account"
+            },
+            "privileges": {
+              "key": "user-account.extensions.x-accessor-ext.group_privileges",
+              "object": "creator-user-account"
+            },
+            "type": {
+              "key": "user-account.extensions.x-accessor-ext.group_type",
+              "object": "creator-user-account"
+            },
+            "uid": {
+              "key": "user-account.extensions.x-accessor-ext.group_uid",
+              "object": "creator-user-account"
+            }
+          },
+          "name": {
+            "key": "user-account.display_name",
+            "object": "creator-user-account"
+          },
+          "org_uid": {
+            "key": "user-account.extensions.x-accessor-ext.org_uid",
+            "object": "creator-user-account"
+          },
+          "session_uid": {
+            "key": "user-account.extensions.x-accessor-ext.session_uid",
+            "object": "creator-user-account"
+          },
+          "session_uuid": {
+            "key": "user-account.extensions.x-accessor-ext.session_uuid",
+            "object": "creator-user-account"
+          },
+          "type": {
+            "key": "user-account.extensions.x-accessor-ext.type",
+            "object": "creator-user-account"
+          },
+          "type_id": {
+            "key": "user-account.extensions.x-accessor-ext.type_id",
+            "object": "creator-user-account"
+          },
+          "uid": {
+            "key": "user-account.user_id",
+            "object": "creator-user-account"
+          },
+          "uuid": {
+            "key": "user-account.extensions.x-accessor-ext.uuid",
+            "object": "creator-user-account"
+          }
+        },
+        "desc": {
+          "key": "file.extensions.x-ocsf-file-ext.description",
+          "object": "file"
+        },
+        "fingerprints": {
+          "algorithm": {
+            "key": "file.extensions.x-ocsf-file-ext.algorithm",
+            "object": "file"
+          },
+          "algorithm_id": {
+            "key": "file.extensions.x-ocsf-file-ext.algorithm_id",
+            "object": "file"
+          },
+          "value": {
+            "key": "file.extensions.x-ocsf-file-ext.algorithm_value",
+            "object": "file"
+          }
+        },
+        "is_system": {
+          "key": "process.extensions.x-ocsf-process-ext.is_system",
+          "object": "process"
+        },
+        "mime_type": {
+          "key": "process.mime_type",
+          "object": "process"
+        },
+        "modified_time": {
+          "key": "process.modified",
```

Comment:
I can't find a modified property on the process in 2.1 https://docs.oasis-open.org/cti/stix/v2.1/os/stix-v2.1-os.html#_hpppnm86a1jm

Reply:
right. i mixed this up with file object